src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.jorge;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+    private static final int NUMBER_5 = 5;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_10 = 10;
+
+    /**
+     *
+     */
+    public FizzBuzz() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)